src/components/Detail.js

```diff
@@ -0,0 +1,37 @@
+import React, {Component} from "react";
+
+const imageUrl = "images/peter-wendt--r5KSMkyoSc-unsplash.jpg";
+
+export default class Details extends Component {
+
+    render() {
+        const {blocks} = this.props;
+        const {city, name, id} = this.props.match.params;
+
+        const block = blocks.filter(block => block.name.toLowerCase() === name.toLowerCase())[0];
+        console.log(block);
+        const market_details = block.markets.filter(market => market.id == id)[0];
```

Comment:
please use `camelCase` for javascript variables. This one should be `marketDetails`